subt/main.py

```diff
@@ -98,6 +85,8 @@ def __init__(self, config, bus):
         self.time = None
         self.max_speed = config['max_speed']
         self.max_angular_speed = math.radians(45)
+        self.dist = config['dist']
```

Comment:
I still have problem with this `dist` :( ... everywhere else I use "dist" as "distance traveled", so I would prefer much more some other variable "safety_dist", "safety_distance", "wall_dist"??